test/built-ins/Set/prototype/union/converts-negative-zero.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2023 Anthony Frehner. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: pending
+description: Set.prototype.union converts -0𝔽 to +0𝔽
+info: |
+    7.b.ii. If nextValue is -0𝔽, set nextValue to +0𝔽.
+features: [Set-methods]
+includes: [compareArray.js]
+---*/
+
+const s1 = new Set([1]);
+const s2 = new Set([-0]);
```

Comment:
The `Set` constructor (through `add`) already normalizes this to positive zero, so nothing in this test would fail if `union` was wrong.